### DISCUSSION TITLE
Opened to pull request?

### DISCUSSION THREAD

**Discussion by aoi-ue** [galojix/roster-wizard#20] [General]
Hey there, I see that this repo is pretty active and would like to contribute to an application that is HRMS (human resource management system) related - are you open to contributions? let me know 😸

**Comment by gregcowell** [comment]
Hi, if you believe you have something valuable to contribute then you are certainly welcome to submit a pull request. Roster Wizard is almost feature complete and already able to perform its intended purpose. The only significant features missing are internationalisation and detailed documentation. You are welcome to attempt adding internationalisation, create documentation or submit bug fixes. You should first make sure you can install the application and use its features successfully. After installation, the user guide has the recommended next steps. To contribute to the code base, you will need to be familiar with Python and Django. More advanced contributions would require knowledge of Google OR-Tools.
